modules/Story/Story.tsx

```diff
@@ -6,12 +6,30 @@ type Props = {
     story: ExtendedStory;
 };
 
-const Story: FunctionComponent<Props> = ({ story }) => (
-    <article>
-        <h2>{story.title}</h2>
-        <h3>{story.subtitle}</h3>
-        <SlateRenderer nodes={JSON.parse(story.content)} />
-    </article>
-);
+const Story: FunctionComponent<Props> = ({ story }) => {
+    if (!story) {
+        return <></>;
+    }
+
+    const {
+        title, subtitle, content, format_version, htmlContent,
+    } = story;
+
+    return (
+        <article>
+            <h2>{title}</h2>
+            <h3>{subtitle}</h3>
+            <SlateRenderer nodes={JSON.parse(content)} />
+
+            {format_version === 1 && (
+                // eslint-disable-next-line react/no-danger
+                <div dangerouslySetInnerHTML={{ __html: htmlContent }} />
+            )}
+            {format_version === 3 && (
+                <SlateRenderer nodes={JSON.parse(content)} />
+            )}
```

Comment:
`<SlateRenderer>` is called two times here, is this intentional?

Reply:
Also, `JSON.parse` can be quite heavy with big objects, maybe move it to a `useMemo`?